api/src/test/scala/io/phdata/rest/WorkspaceControllerSpec.scala

```diff
@@ -99,6 +100,7 @@ class WorkspaceControllerSpec
       .returning(OptionT.some(
         WorkspaceMemberEntry(standardUserDN.value, name, Some("johndoe@phdata.io"), List.empty, List.empty, List.empty, List.empty)
       ))
+    (impalaService.invalidateMetadata(_: Long)(_: AppContext[IO])(_: Sync[IO])).expects(123L, context, *).returning(IO.unit)
```

Comment:
Any idea what  the extra `*` is for? `expects(123L, context, *)`

Reply:
It's for `Sync[IO]`